src/tests/tribol_comp_geom.cpp

```diff
@@ -510,11 +510,16 @@ TEST_F( CompGeomTest, codirectional_normals_3d )
    int numVerts = 4;
    int numCells = 2;
    int lengthNodalData = numCells * numVerts;
-   real element_thickness = 1.;
+   real element_thickness[numCells];
    real x[lengthNodalData];
    real y[lengthNodalData];
    real z[lengthNodalData];
```

Comment:
I think the `int` variables need to be `const` or `constexpr` or else the arrays have variable-length, which is non-standard

Reply:
I added constexpr to these ints and elsewhere as required in this test. I won't address this in all tests in this PR.